.travis.yml

```diff
@@ -14,7 +14,11 @@
 language: java
 jdk:
   - oraclejdk8
-script: mvn -U verify javadoc:jar
+env:
+  - EXTRA_OPTS=
```

Comment:
why set `EXTRA_OPTS`?  Why not just set `mvn -U clean verify javadoc:jar -Daccumulo.version=1.8.1`?

Reply:
This creates a Matrix build in Travis. Travis will run all combinations of `env` and `script`. This gives us two builds... one with the default (1.7.3) and one with 1.8.1 explicitly set.